hue/hue/base/kustomization.yaml

```diff
@@ -0,0 +1,133 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - hive-site-xml-secret.yaml
+  - hue-db-init-job.yaml
+  - hue-dc.yaml
+  - hue-ini-secret.yaml
+  - hue-mysql-dc.yaml
+  - hue-mysql-pvc.yaml
+  - hue-mysql-secret.yaml
+  - hue-mysql-service.yaml
+  - hue-mysqld-exporter-secret.yaml
+  - hue-route.yaml
+  - hue-secret.yaml
+  - hue-service.yaml
+
+commonLabels:
+  opendatahub.io/component: "true"
+  component.opendatahub.io/name: hue
+  component.opendatahub.io/part-of: datacatalog
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+  - name: hue-config
+    env: params.env
+
+vars:
+  - name: namespace
+    objref:
+      kind: Service
+      apiVersion: v1
+      name: hue
+    fieldref:
+      fieldpath: metadata.namespace
+  - name: storage_class
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: hue-config
+    fieldref:
+      fieldpath: data.storage_class
+  - name: s3_endpoint
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: hue-config
+    fieldref:
+      fieldpath: data.s3_endpoint
+  - name: s3_region
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: hue-config
+    fieldref:
+      fieldpath: data.s3_region
+  - name: s3_is_secure
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: hue-config
+    fieldref:
+      fieldpath: data.s3_is_secure
+  - name: s3_access_key_id
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: hue-secret
+    fieldref:
+      fieldpath: stringData.s3_access_key_id
+  - name: s3_secret_access_key
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: hue-secret
+    fieldref:
+      fieldpath: stringData.s3_secret_access_key
+  - name: database_user
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: hue-mysql-secret
+    fieldref:
+      fieldpath: stringData.database-user
+  - name: database_password
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: hue-mysql-secret
+    fieldref:
+      fieldpath: stringData.database-password
+  - name: database_root_password
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: hue-mysql-secret
+    fieldref:
+      fieldpath: stringData.database-root-password
+  - name: database_name
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: hue-mysql-secret
+    fieldref:
+      fieldpath: stringData.database-name
+
+configurations:
+  - params.yaml
+
+images:
+  - name: hue
+    newName: shgriffi/hue
+    newTag: test
+  - name: mysql
+    newName: openshift/mysql
+    newTag: "5.7"
+  - name: mysqld-exporter
+    newName: prom/mysqld-exporter
+
+patchesJson6902:
+  - patch: |-
```

Comment:
Ah, you can partially disregard my comment about the image above:)

But, the problem with using images from `openshift` namespace is that versions disappear between OCP version releases - we hit it every now and then with JupyterHub DB. I would rather us to have control over when we update, then disappearing image forcing us to - can we rather pull the image from RH container catalog?

Reply:
:) Sure, we can change the base image.

Reply:
removed since `Deployment`s don't support triggers anyways (changed from `DeploymentConfig`